crates/compiler/can/tests/test_suffixed.rs

```diff
@@ -26,17 +26,14 @@ mod suffixed_tests {
     ```roc
     main =
         line! "Ahoy"
-        {} = "There" |> Stdout.line!
+        _ = "There" |> Stdout.line!
 
         Task.ok {}
 
     main =
-        Task.await [line "Ahoy"] \{} ->
-            Task.await [Stdout.line "there"] \{} ->
+        Task.await [line "Ahoy"] \_ ->
```

Comment:
I think this comment it now incorrect.

Reply:
good catch. however the pr is not yet ready for review. I'll update the tests as the last step

Reply:
nw, my bad

Reply:
I moved most fixes unrelated to types annotations here: https://github.com/roc-lang/roc/pull/6888